numbat-cli/src/main.rs

```diff
@@ -133,27 +133,12 @@ impl Cli {
             Config::default()
         };
 
-        config.main.load_prelude = if args.no_prelude {
-            false
-        } else {
-            config.main.load_prelude
-        };
-        config.main.load_user_init = if args.no_prelude || args.no_init {
-            false
-        } else {
-            config.main.load_user_init
-        };
+        config.main.load_prelude &= !args.no_prelude;
+        config.main.load_user_init &= !(args.no_prelude || args.no_init);
```

Comment:
Thank you. I knew there was a better way to write this, I was just too lazy to think yesterday :smile: